src/components/editor-page/document-bar/revisions/revision-list-entry.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { DateTime } from 'luxon'
+import React, { useCallback, useMemo } from 'react'
+import { ListGroup } from 'react-bootstrap'
+import { Trans, useTranslation } from 'react-i18next'
+import { ForkAwesomeIcon } from '../../../common/fork-awesome/fork-awesome-icon'
+import { UserAvatar } from '../../../common/user-avatar/user-avatar'
+import styles from './revision-list-entry.module.scss'
+import type { RevisionMetadata } from '../../../../api/revisions/types'
+import { getUserDataForRevision } from './utils'
+import { showErrorNotification } from '../../../../redux/ui-notifications/methods'
+import { useAsync } from 'react-use'
+import { ShowIf } from '../../../common/show-if/show-if'
+import { WaitSpinner } from '../../../common/wait-spinner/wait-spinner'
+
+export interface RevisionListEntryProps {
+  active: boolean
+  onSelect: (selectedId: number) => void
+  revision: RevisionMetadata
+}
+
+/**
+ * Renders an entry in the revision list.
+ * @param active true if this is the currently selected revision entry.
+ * @param onSelect Callback that is fired  when this revision entry is selected.
+ * @param revision The metadata for this revision entry.
+ */
+export const RevisionListEntry: React.FC<RevisionListEntryProps> = ({ active, onSelect, revision }) => {
+  useTranslation()
+
+  const onSelectRevision = useCallback(() => {
+    onSelect(revision.id)
+  }, [revision, onSelect])
+
+  const revisionCreationTime = useMemo(() => {
+    return DateTime.fromISO(revision.createdAt).toFormat('DDDD T')
+  }, [revision.createdAt])
+
+  const revisionAuthors = useAsync(async () => {
+    // TODO This is currently not implemented in the backend and therefore hardcoded.
```

Comment:
Is this still the case? If so, is there an issue or pull request that can be linked?

Reply:
I've missed this.
Changed.